teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -9,7 +9,8 @@
     "Authors:\n",
```

Comment:
<p>Mention why there are so many services for the same functionality (historical reasons, mainly) and how we will focus on the REST variant (and link to the T011 notebook for reference).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done</p>